pulp_container/app/serializers.py

```diff
@@ -132,6 +140,37 @@ class Meta:
         model = models.Blob
 
 
+class ConfigBlobSerializer(NoArtifactContentSerializer):
+    """
+    Serializer for Blob Config Manifest
+    """
+
+    architecture = serializers.CharField(
+        help_text="The CPU architecture which the binaries in this image are built to run on"
+    )
+    os = serializers.CharField(
+        help_text="The name of the operating system which the image is built to run on"
```

Comment:
Let's wrap the help messages around with the `gettext` call. This enables others to localize Pulp.

```
help_text=_("blabla
```